index.html

```diff
@@ -54,5 +56,249 @@ <h1 class="name">
+          <div>
+            <ul class="tech-Stack">
+              <li id="html">HTML</li>
+              <li id="css">CSS</li>
+              <li id="js">JAVASCRIPT</li>
+            </ul>
+          </div>
+    
+          <!--  -->
+          <div><button type="button" class="seeProject">see project</button></div>
+    
+        </div>
+    </div>
+
+    <!-- work tree -->
+    <div  class="workCard1"> 
+        <div class="block1">
+            <img src="images/work3 (1).png" alt="">
+        </div>
+    <!-- details block -->
+        <div class="block2">
+            <!--  -->
+            <h2 class="tonic">Tonic</h2>
+            
+            <div class="tools">
+                <h4 class="canopy">CANOPY</h4>
+                <img src="./images/Counter.png" alt="counter" />
+                <h4 class="counter-shadow">Back End Dev</h4>
+                <img src="./images/Counter.png" alt="counter" />
+                <h4 class="counter-shadow">2015</h4>
+              </div>
+          <!--  -->
+    
+          <div class="project-details">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <!--  -->
+    
+          <div>
+            <ul class="tech-Stack">
+              <li id="html">HTML</li>
+              <li id="css">CSS</li>
+              <li id="js">JAVASCRIPT</li>
+            </ul>
+          </div>
+    
+          <!--  -->
+          <div><button type="button" class="seeProject">see project</button></div>
+    
+        </div>
+    </div>
+
+     <!-- work four -->
+     <div  class="workCard1"> 
+        <div class="block1">
+            <img src="images/work4.png" alt="">
+        </div>
+    <!-- details block -->
+        <div class="block2">
+            <!--  -->
+            <h2 class="tonic">Tonic</h2>
+            
+            <div class="tools">
+                <h4 class="canopy">CANOPY</h4>
+                <img src="./images/Counter.png" alt="counter" />
+                <h4 class="counter-shadow">Back End Dev</h4>
+                <img src="./images/Counter.png" alt="counter" />
+                <h4 class="counter-shadow">2015</h4>
+              </div>
+          <!--  -->
+    
+          <div class="project-details">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <!--  -->
+    
+          <div>
+            <ul class="tech-Stack">
+              <li id="html">HTML</li>
+              <li id="css">CSS</li>
+              <li id="js">JAVASCRIPT</li>
+            </ul>
+          </div>
+    
+          <!--  -->
+          <div><button type="button" class="seeProject">see project</button></div>
+    
+        </div>
+    </div>
+</section>
+
+<!-- about me section -->
+
+<section class="about-me">
+
+    <div class="myself">
+        <div class="">

```

Comment:
- [OPTIONAL] Since you aren't giving this a class name, I will advise you to remove it.